scripts/ci/test_cut_ironclaw_release.py

```
@@ -0,0 +1,108 @@
+            arguments = {
+                "requested_version": VERSION,
+                "requested_sha": SHA,
+                "manifest_version": VERSION,
+                "checked_out_sha": SHA,
+                "get_tag_target": lambda _tag: None,
+                "create_tag": lambda _tag, _sha: None,
+            }
+            arguments[field] = value
+            with (
+                self.subTest(field=field),
+                self.assertRaisesRegex(release.ReleaseTagError, message),
```

Ruff (0.16.0)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)